vault/resource_raft_snapshot_agent_config_test.go

```diff
@@ -0,0 +1,116 @@
+package vault
+
+import (
+	"testing"
+
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccRaftSnapshotAgentConfig_basic(t *testing.T) {
+	name := acctest.RandomWithPrefix("tf-test-raft-snapshot")
+	resource.Test(t, resource.TestCase{
+		Providers:    testProviders,
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
We should update the PreCheck to use `util.TestEntPreCheck(t)` instead. This includes the `accPreCheck` checks, but also ensures that these tests only run on Vault Enterprise versions

Reply:
✅ Done